articles/other/other-ref-browsers.md

```diff
@@ -23,28 +23,19 @@ This document outlines the browser requirement for UKCloud services.
 
 ## UKCloud for VMware
 
-For UKCloud for VMware, we recommend that you use the Mozilla Firefox Extended Support Release (ESR 52) 32-bit browser. This browser is supported by both the UKCloud Portal and by the Web Console for vCloud Director 8.20 (the version used by UKCloud for VMware).
+For UKCloud for VMware, vCloud Director 9.1 is compatible with many current and last major browser revisions.
+Version 9.1 of vCloud Director was tested by VMware and supports:
 
-Firefox ESR 52 is an older version of Firefox, updated with new security patches, that remains compatible with the plugin type that the vCloud Director Web Console uses on our platform. You can find Firefox ESR at:
+- Google Chrome 64
+- Mozilla Firefox 58
+- Microsoft Edge 41
+- Microsoft Internet Explorer 11
 
-<https://www.mozilla.org/en-US/firefox/organizations/>
+>Note: Flash must be enabled in the browser to access the vCloud Director Web Console.
 
-> [!NOTE]
-> Make sure you download ESR 52, which is the last ESR release to support legacy add-ons. Support for ESR 52 ends in August 2018. We plan to upgrade UKCloud for VMware to vCloud Director 9.1, which will eliminate the need for the Web Console plugin. For more information, see <https://blog.mozilla.org/addons/2017/10/03/legacy-add-on-support-on-firefox-esr/>. You should also make sure that you download the 32-bit version of the browser.
+>Note: Edge does not support plugins, so functions such as console redirection and OVF upload using the legacy UI do not work with Edge.
```

Comment:
Use correct syntax for notes.
See: https://docs.ukcloud.com/articles/other/other-ref-knowledge-guidelines.html#for-notes